### PR TITLE
Reword unrealistic recommendation

### PR DESCRIPTION
**Pull request by cynthia**
Resolves some part of #5.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,7 +72,7 @@
 
 The TAG considers each new technology proposal *both* individually, *and* as they fit together with the web platform as a whole. The web must be cross-platform, so multi-stakeholder (in particular, multi-browser) support for privacy-related specifications is essential if they are going to achieve the goal of increasing privacy on the web. When we consider whether something makes the web platform better, we should be explicit about what the baseline for comparison is. Is a proposal better for privacy when compared to usage of third-party cookies? Or when compared with a web free from third-party cookies altogether? What about when some user agents restrict third-party cookies, but others do not?
 
-We urge that the deprecation of third-party cookies continues across the web platform irrespective of the progress of proposals which seek to replace some of the functionality of third-party cookies.
+We want to emphasize that as the proposals progress, implementations should have a strong commitment and deadline toward deprecating third-party cookies.
```
Comment (plinss) [inline index.html]:
```suggestion
We want to emphasize that as the replacement proposals progress, implementations should have a strong commitment toward, and reasonable time frame for, deprecating third-party cookies.  
```

File: index.html [hunk 2]
```diff
@@ -72,12 +72,12 @@
...
 
 Many varied proposals are being incubated in W3C Community Groups (eg. [PATCG](https://patcg.github.io/), [Privacy CG](https://github.com/privacycg/), [WICG](https://wicg.io/)) as well as outside (eg. [Privacy Sandbox](https://www.privacysandbox.com/)), and in these incubation stages multi-stakeholder support, consensus, and possible timelines for standardisation are uncertain, and far from guaranteed.
 
-We are also wary of new approaches being introduced while third-party cookies are still available to authors, as this comes with a risk that more, parallel channels for privacy invasion become possible, rather than fewer. 
-Thus, the deprecation of third-party cookies should not be dependent upon the standardization of replacement technologies.
+We are also wary of new approaches being introduced while third-party cookies are still available to authors, as this comes with a risk that more, parallel channels for privacy invasion become possible, rather than fewer.
+Given this context, there is an urgency to have a strict timeline for deprecation - which also means that the alternatives must materialize before this happens.
```
Comment (hober) [inline index.html]: I think this overstates things. Basically, I'd drop the bit after the hyphen.

**Review by hober — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -72,12 +72,12 @@
...
```
Comment (cynthia) [inline index.html]:
```suggestion
Given this context, there is an urgency to have a strict timeline for deprecation.
```

File: index.html [hunk 4]
```diff
@@ -72,12 +72,12 @@
...
 
 Many varied proposals are being incubated in W3C Community Groups (eg. [PATCG](https://patcg.github.io/), [Privacy CG](https://github.com/privacycg/), [WICG](https://wicg.io/)) as well as outside (eg. [Privacy Sandbox](https://www.privacysandbox.com/)), and in these incubation stages multi-stakeholder support, consensus, and possible timelines for standardisation are uncertain, and far from guaranteed.
 
-We are also wary of new approaches being introduced while third-party cookies are still available to authors, as this comes with a risk that more, parallel channels for privacy invasion become possible, rather than fewer. 
-Thus, the deprecation of third-party cookies should not be dependent upon the standardization of replacement technologies.
+We are also wary of new approaches being introduced while third-party cookies are still available to authors, as this comes with a risk that more, parallel channels for privacy invasion become possible, rather than fewer.
+Given this context, there is an urgency to have a strict timeline for deprecation.
```
Comment (cynthia) [inline index.html]:
```suggestion
Given this context, we see an urgency to have a strict timeline for deprecation.
```

**Review by torgo — APPROVED** [review]
lgtm

**Review by hober — APPROVED** [review]

**Pull request merged**